index.ts

```diff
@@ -198,6 +199,34 @@ async function increaseDissolveDelay(
   ok();
 }
 
+async function setDissolveDelay(
+  neuronId: bigint,
+  years: number,
+  days: number,
+  minutes: number,
+  seconds: number
+) {
+  const identity = await getLedgerIdentity();
+  await assertLedgerVersion({ identity, minVersion: "2.1.7" });
+  const governance = GovernanceCanister.create({
+    agent: await getAgent(identity),
+    hardwareWallet: false,
+  });
+
+  const dissolveDelaySeconds =
+    years * SECONDS_PER_YEAR +
+    days * SECONDS_PER_DAY +
+    minutes * SECONDS_PER_MINUTE +
+    seconds;
+
+  await governance.setDissolveDelay({
+    neuronId: neuronId,
```

Comment:
you have prettier in the repo? `neuronId: neuronId` -> `neuronId` can be auto formatted I think no?

Reply:
Yes, prettier is set up. I'll take a look at it after this PR.